apps/sparo-lib/src/services/SparoProfileService.ts

```diff
@@ -227,10 +227,14 @@ ${availableProfiles.join(',')}
    */
   public async syncProfileState({
```

Comment:
If we use sparo profile in checkout, the profile will be recorded, and after users checkout to another branch, they will use the same profile in that branch.

Should we recored from/to selector here?

Reply:
good one! let update it accordingly